src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -231,12 +232,25 @@ private OffenderLatestRecall getOffenderLatestRecall(Optional<Long> maybeOffende
     public CommunityOrPrisonOffenderManager allocatePrisonOffenderManagerByNomsNumber(final @PathVariable String nomsNumber,
                                                                  final @RequestBody CreatePrisonOffenderManager prisonOffenderManager) {
         log.info("Request to allocate a prison offender manager to {} at prison with code {}", nomsNumber, prisonOffenderManager.getNomsPrisonInstitutionCode());
+
+        String errorMessage = prisonOffenderManager.validate();
+        if (!errorMessage.isBlank()) {
+            throw new InvalidAllocatePOMRequestException(prisonOffenderManager, errorMessage);
+        }
+
         return Optional.ofNullable(prisonOffenderManager.getOfficerCode())
                 .map(staffCode -> offenderManagerService.allocatePrisonOffenderManagerByStaffCode(nomsNumber, staffCode, prisonOffenderManager))
                 .orElseGet(() -> offenderManagerService.allocatePrisonOffenderManagerByName(nomsNumber, prisonOffenderManager))
                 .orElseThrow(() -> new NotFoundException(String.format("Offender with noms number %s not found", nomsNumber)));
     }
 
+    public static class InvalidAllocatePOMRequestException extends BadRequestException {
+        InvalidAllocatePOMRequestException(CreatePrisonOffenderManager createPrisonOffenderManager, String message) {
+            super(message);
+            log.info("Bad request: " + createPrisonOffenderManager);
```

Comment:
Maybe bump this as a warning since it indicates we have an API client doing bad things